### ISSUE TITLE
MILLION_RPU unit isn't supported by provider

### ISSUE THREAD

**Issue by tcarrondo** [mongodb/terraform-provider-mongodbatlas#854]
### Terraform CLI and Terraform MongoDB Atlas Provider Version

```hcl
$ terraform version
Terraform v1.3.0
on linux_amd64
+ provider registry.terraform.io/mongodb/mongodbatlas v1.4.6
```

### Terraform Configuration File 

```hcl
resource "mongodbatlas_alert_configuration" "with_thresholds" {

  project_id = data.mongodbatlas_project.main.id
  event_type = "OUTSIDE_SERVERLESS_METRIC_THRESHOLD"
  enabled    = true

  notification {
    type_name    = "TEAM"
    interval_min = 5
    delay_min    = 120
    team_id      = data.mongodbatlas_teams.main.team_id
  }

  metric_threshold_config {
    metric_name = "SERVERLESS_TOTAL_READ_UNITS"
    operator    = "GREATER_THAN"
    threshold   = 1
    units       = "MILLION_RPU"
    mode        = "AVERAGE"
  }
}
```

### Steps to Reproduce

1. `terraform init`
2. `terraform apply`

### Expected Behavior

Resource should be created 

### Actual Behavior

```
Error: expected metric_threshold_config.0.units to be one of [RAW BITS BYTES KILOBITS KILOBYTES MEGABITS MEGABYTES GIGABITS GIGABYTES TERABYTES PETABYTES MILLISECONDS SECONDS MINUTES HOURS DAYS], got MILLION_RPU
```

**Comment by martinstibbe** [comment]
@tcarrondo Current API Docs and codebase reflect this list as supported values Atlas portal does show this setting as an available value will verify with engineering in case we are missing other available values

* `threshold` - Threshold value outside of which an alert will be triggered.
* `units` - The units for the threshold value. Depends on the type of metric.
    Accepted values are:
      - `RAW`
      - `BITS`
      - `BYTES`
      - `KILOBITS`
      - `KILOBYTES`
      - `MEGABITS`
      - `MEGABYTES`
      - `GIGABITS`
      - `GIGABYTES`
      - `TERABYTES`
      - `PETABYTES`
      - `MILLISECONDS`
      - `SECONDS`
      - `MINUTES`
      - `HOURS`
      - `DAYS`

**Comment by martinstibbe** [comment]
Internal Engineering tracking ticket [INTMDB-406](https://jira.mongodb.org/browse/INTMDB-406)

**Comment by akierstein-insider** [comment]
+1

**Comment by Zuhairahmed** [comment]
@tcarrondo @akierstein-insider  we have just released v1.5.0, can you let us know if this resolves issue?
